core/shared/src/main/scala/cats/effect/IORuntime.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2020 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cats.effect
+
+import scala.concurrent.ExecutionContext
+import java.util.concurrent.atomic.AtomicInteger
+import java.util.concurrent.Executors
+
+case class IORuntime(compute: ExecutionContext, timer: UnsafeTimer) extends IORuntimePlatform {
```

Comment:
`final` Also, *should* it be a case class? Or should it just hide the `compute` and `timer` instances?

Reply:
Fixed